src/sentry/static/sentry/app/components/events/interfaces/threads/threadSelector/option.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import {css} from '@emotion/core';
+
+import {t} from 'app/locale';
+import Tooltip from 'app/components/tooltip';
+import TextOverflow from 'app/components/textOverflow';
+import {EntryTypeData} from 'app/types';
+import {IconFire, IconCheckmark} from 'app/icons';
+
+import {Grid, GridCell} from './styles';
+
+type Props = {
+  id: string;
+  details: ThreadInfo;
+  crashedInfo?: EntryTypeData;
+  name?: string;
+  crashed?: boolean;
+};
+
+type ThreadInfo = {
+  label?: string;
+  filename?: string;
+};
+
+const Option = ({id, name = t('<unknown>'), details, crashed, crashedInfo}: Props) => {
+  const {label = t('<unknown>'), filename = t('<unknown>')} = details;
+  return (
+    <Grid>
+      <GridCell>
+        <Id>
+          <Tooltip title={`#${id}`} position="top">
+            <TextOverflow>{`#${id}`}</TextOverflow>
+          </Tooltip>
+        </Id>
+      </GridCell>
+      <GridCell>
+        <Name>
+          <Tooltip title={name} position="top">
+            <TextOverflow>{name}</TextOverflow>
+          </Tooltip>
+        </Name>
+      </GridCell>
+      <GridCell>
+        <Label>
+          <Tooltip title={label} position="top">
+            <TextOverflow>{label}</TextOverflow>
+          </Tooltip>
+        </Label>
+      </GridCell>
+      <GridCell>
+        <FileName>
+          <Tooltip title={filename} position="top">
+            <TextOverflow>{filename}</TextOverflow>
+          </Tooltip>
+        </FileName>
+      </GridCell>
+      <GridCell>
+        <Icon>
+          {crashed ? (
+            crashedInfo ? (
+              <Tooltip
+                title={`${t('errored with')} ${crashedInfo.values[0].type}`}
```

Comment:
You should use a %s placeholder or tct() here. Concatenating the values together makes translation hard.